### PR TITLE
Validate: Add URL inclusion

### PR DESCRIPTION
**Pull request by mmase**
@mikesherov @Attamusc @weikinhuang

### CODE REVIEW

**Comment by mmase** [comment]
FYI, the reason I can't use the already existing `Url` validation is because it tests beginning of line

File: util/validate.js [hunk 1]
```diff
@@ -129,6 +129,10 @@ define(function() {
       test: test.bind(/^((?:https?):\/\/)?(?:(?:(?:[\w\.\-\+!$&\'\(\)*\+,;=_]|%[0-9a-f]{2})+:)*(?:[\w\.\-\+%!$&\'\(\)*\+,;=]|%[0-9a-f]{2})+@)?(?:[A-Za-z0-9_\-]+\.)(?:[A-Za-z0-9\-\._])+(?::\d+)?(?:[\/|\?](?:[\w#!:\.\?\+=&@$\'~*,;_\/\(\)\[\]\-]|%[0-9a-f]{2})*)?$/),
       message: "This field must be a valid URL"
     },
+    UrlIncl:{
+      test: test.bind(/((https?:\/\/|www\.)[^\s<]{3,})/i),
```
Comment (weikinhuang) [inline util/validate.js]: why `[^\s<]{3,}`?

Comment (mmase) [inline util/validate.js]: No clue, copied from Poet.

Comment (mmase) [inline util/validate.js]: I can use the above `/^(http(?:s)?:\/\/|www.)[^<>]*$/` and remove the `^`

Comment (weikinhuang) [inline util/validate.js]: might be a good idea, remove `^` `$` and add `\s` to the exclude grouping

**Comment by mmase** [comment]
@weikinhuang @mikesherov @Attamusc updated!

**Comment by weikinhuang** [comment]
:+1:

File: test/specs/util/validate.js [hunk 2]
```diff
@@ -61,6 +61,11 @@ define(['util/validate'], function(validate) {
     Html: {
       good: ['<p></p>', '<em></em>', '<u></u>', '<strong></strong>'],
       bad: ['<script></script>', '<embed></embed>']
+    },
+
+    UrlIncl: {
```
Comment (Attamusc) [inline test/specs/util/validate.js]: This name feels weird...

Comment (mmase) [inline test/specs/util/validate.js]: Open to suggestions! Any ideas?

Comment (mmase) [inline test/specs/util/validate.js]: @Attamusc `ContainsUrl`?

Comment (mikesherov) [inline test/specs/util/validate.js]: Much better.

**Comment by mikesherov** [comment]
Once name change...

**Comment by mikesherov** [comment]
:+1:

**Pull request merged**